deployments/multiple-ragapps/data/traefik/traefik.yml

```diff
@@ -5,10 +5,25 @@ api:
 providers:
   docker:
     exposedByDefault: false
+  file:
+    filename: /opt/traefik/config/dynamic.yml
+    watch: true
 entryPoints:
-  http:
+  web:
     address: ":80"
+  websecure:
+    address: ":443"
+    http:
+      tls: true
 
+certificatesResolvers:
+  letsencrypt:
+    acme:
+      # Please change your email in the production environment
```

Comment:
as no one is reading this, what happens if we don't do this? 
maybe force setting it via .env ?

Reply:
> what happens if we don't do this?

I've a test when not specifying the email, it worked - but it could be a cache on acme.json file.

> maybe force setting it via .env ?

I'll check, if traefik support env for config file then we should force to specify the mail.

Reply:
@leehuwuj what about this?